### ISSUE TITLE
Bug: Handle comments in comma-spacing

### ISSUE THREAD

**Issue by alberto** [eslint/eslint#4389]
Right now, `comma-spacing` doesn't correctly handle comments.

This should not error:
```js
myfunc(404, true /* bla bla bla */, 'hello');
```

And this should be valid:
```js
        {
            code: "myfunc(404, true/* bla bla bla */ /* hi */, 'hello');",
            output: "myfunc(404, true/* bla bla bla *//* hi */, 'hello');",
            errors: [
                {
                    message: "There should be no space before ','.",
                    type: "Punctuator"
                }
            ]
        }
```

**Comment by eslintbot** [comment]
Thanks for the issue! If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.
